composer.json

```
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.1",
```

```suggestion
        "php": ">=7.2",
```
 (If it supports PHP 7.2, it likely supports PJP 8.0 but too)